packages/botfuel-dialog/src/dialogs/prompt-dialog.js

```diff
@@ -236,24 +291,39 @@ class PromptDialog extends Dialog {
   async execute(adapter, userMessage, messageEntities) {
     logger.debug('execute', userMessage, messageEntities);
     const userId = userMessage.user;
-    const previouslyMatchedEntities =
-      (await this.brain.conversationGet(userId, this.parameters.namespace)) || {};
+
+    const dialogCache = await this.brain.conversationGet(userId, this.parameters.namespace);
+    const previouslyMatchedEntities = (dialogCache && dialogCache.entities) || {};
```

Comment:
according to our convention (see doc about brain), it should be _entities and _question

Reply:
ok will change this

Reply:
done